perfkitbenchmarker/virtual_machine.py

```diff
@@ -108,9 +97,8 @@ def __init__(self, vm_spec):
     """
     super(BaseVirtualMachine, self).__init__()
     with self._instance_counter_lock:
-      self.name = 'perfkit-%s-%d' % (FLAGS.run_uri, self._instance_counter)
+      self.name = 'pkb-%s-%d' % (FLAGS.run_uri, self._instance_counter)
```

Comment:
Sure - I'll put it off till the next PR. The issue is that on Azure they use the vm name as the username and they have a length restriction of 15 characters for the username.